src/style_guideline/reloading_message.md

```diff
@@ -2,4 +2,44 @@
 
 You **must not** send an automatic message when the world is reloaded. It will clutter the chat unnecessarily and create a bad experience for the users.
 
-Most of the time, the message will be an installation message, in which case you can use [Datapack Advancement](../conventions/datapack_advancement.md) instead.
\ No newline at end of file
+Most of the time, the message will be an installation message, in which case you can use [Datapack Advancement](../conventions/datapack_advancement.md) instead.
+
+> **Example:** Use `load.json` call `setup` or `init` mcfunction that has tellraw command.
+```json
+{
+    "values": [
+        "convention:datapack/setup",
+    ]
+}
+```
+``convention:datapack/setup``
+```mcfunction
+tellraw @a <...>
+```
```

Comment:
This message conveyed the *opposite* of what to do to the reader

Reply:
This message isn't necessary, right?